info.json

```diff
@@ -9,32 +9,44 @@
             "name": "sOARFramework",
             "type": "solutionpack",
             "label": "SOAR Framework",
-            "version": "1.0.0",
+            "version": "1.1.0",
             "minVersion": null
         }
     ],
     "prerequisite": null,
-    "publisher": "Community",
-    "description": "Playbooks to extract file metadata and content. The result is added to the record's description",
-    "help": null,
+    "publisher": "Fortinet SE",
+    "description": "Playbooks to extract file metadata and content. The result is added to the record's Description. Notify the count of IOCs",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/README.md",
     "category": [
-        "Investigation",
-        "Enrichment"
+        "Enrichment",
+        "Investigation"
     ],
     "supportInfo": null,
     "iconLarge": "fsr-icon-large.png",
     "recordTags": [
         "Utility"
     ],
-    "fsrMinCompatibility": "7.2.1",
-    "date": "2022-09-14T08:28:04+00:00",
+    "fsrMinCompatibility": "7.2.0",
+    "date": "2022-09-14T11:31:51+00:00",
     "contents": {
         "playbooks": [
             {
-                "name": "02 - Use Case - File Content Extraction - 1.1.0"
+                "name": "02 - Use Case - File Content Extraction"
             }
         ],
         "globalVariables": [
+            {
+                "id": 7,
+                "name": "Server_fqhn",
```

Comment:
Global variables are not needed here

Reply:
Removed all Global Variables from info.json